maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/controller/RepOrderController.java

```diff
@@ -210,4 +212,34 @@ public ResponseEntity<Void> updateRepOrder(@PathVariable int repId,
     repOrderService.update(repId, updatedFields);
         return ResponseEntity.ok().build();
     }
+
+    @GetMapping(value = "/eligible-for-fdc-delayed-pickup", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(description = "Retrieve a set of rep order ids that have passed the Final Defence Cost delay period")
+    @ApiResponse(responseCode = "200",
+            content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE)
+    )
+    @StandardApiResponse
+    public ResponseEntity<Object> findEligibleForFdcDelayedPickup(@RequestParam(value = "delay") int delayPeriod,
+                                                             @RequestParam(value = "numRecords") int numRecords,
+                                                             @RequestParam(value = "dateReceived")  @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate dateReceived)
+    {
+        log.info("Get Rep Orders For Fdc Delay Received");
+        List<Integer> repIdList = repOrderService.findEligibleForFdcDelayedPickup(delayPeriod, dateReceived, numRecords);
```

Comment:
`List<Integer> repIdList` should by a `Set` as all Rep Order Ids are unique.

Reply:
Good call, changed across the process.